src/styles/tailwind.css

```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Material+Icons+Outlined');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
```

Comment:
Why?

Reply:
The thumbnail of `dotContent` is not always an image, it can be an icon. In Dotcms, Material Icon is used to display icons. So, in case the image fails, I show the icon.

Reply:
We care about the icon in the backend to identify the content type, in this case we don't care if no image is coming, max we can put a square as a place holder.

Reply:
I got it. I'll make the change.